implementations/scripts/Metric.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+
+
+class Metric():
+    """
+    Create a dataframe from data-list, which is the data
+    fetched by Perceval
+
+    :param data_list: A list of dictionaries.
```

Comment:
Please, use "items" instead of "data_list". See rationale in other comment in this review.

Reply:
Done!